src/payment_import/fio_cz/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "version": "1",
+  "information": {
+    "name": "fio_cz-import",
+    "displayName": "Fio bank payment import",
+    "description": "This plugin enables you to import payments from Fio bank to UCRM. See https://www.fio.cz/bankovni-sluzby/api-bankovnictvi",
+    "url": "https://github.com/Ubiquiti-App/UCRM-plugins",
```

Comment:
this should target the plugin directory, so that plugin readme would be instantly visible